### DISCUSSION TITLE
Generate field value by concatenating firstname and lastname

### DISCUSSION THREAD

**Discussion by clawes** [filamentphp/filament#467] [General]
I have employee provisioning form.  I want the username field to be derived by concatenating the firstname and lastname.
So when the user fills the firstname (John) and lastname (Doe), the username should default to 'john.doe'.  I suspect that the dependable option is tool to do this.  However,  I don't know how to:
- Check that both dependable  fields are filled
- Update the value of the dependent (username) field.

Thanks for your response.

**Comment by bilal-pixbit** [comment]
Tables\Columns\TextColumn::make('name')
                    ->getStateUsing(fn ($record) => $record->attendee->last_name ?  $record->attendee->first_name . ' ' . $record->attendee->last_name :  $record->attendee->first_name )
                    ->searchable(),
              
                    
                    try this
